dashboard-demo.html

```diff
@@ -23,6 +23,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.js"></script>
     <script src="js/dashboard-demo/dashboard-updates.js"></script>
     <script src="js/dashboard-demo/dashboard-test-scenarion.js"></script>
+    <script src="js/dashboard-demo/tts.js"></script>
```

Comment:
Please rename the file to sound-system.js, tts is only one of the functions provided